huxley/www/js/components/DelegateSelect.js

```diff
@@ -18,8 +18,7 @@ var DelegateSelect = React.createClass({
     return (
       <select
         onChange={this.props.onChange}
-        value={this.props.selectedDelegateID}
-        disabled>
```

Comment:
Could you remind me why this was disabled in the first place?

Reply:
It was done here: [https://github.com/bmun/huxley/commit/64c84d2a1c24aa65543a221a96ca71fd9bea6089#diff-3bd1cda3536a01c6309345f7a007ca59](url), I assume to disable some functionality before conference. This ties into why I opened #632